extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json

```diff
@@ -0,0 +1,158 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id":
+    "https://ns.adobe.com/experience/campaign/orchestration/reportingevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "User journey reporting event",
+  "type": "object",
+  "description": "",
+  "definitions": {
+    "reportingevent": {
+      "properties": {
+        "xdm:timestamp": {
+          "title": "Timestamp for reporting event",
+          "type": "string",
+          "format": "date-time",
+          "description": "Time of generation of reporting event"
+        },
+        "xdm:dataEnrichmentSource": {
+          "title": "Data Source",
+          "$ref": "https://ns.adobe.com/xdm/data/datasource",
+          "description": "Data source used for data enrichment"
+        },
+        "xdm:dataEntityUID": {
+          "title": "Unique identifier of data entity",
+          "type": "string",
+          "format": "uri",
+          "description":
+            "The identifier for the data entity used for data enrichment"
+        },
+        "xdm:orgID": {
```

Comment:
This looks like it might run into a conflict with #323

Reply:
Talked with the PR raiser , we will handle this before merging.